packages/teams-js/src/private/privateAPIs.ts

```diff
@@ -125,6 +125,7 @@ export function openFilePreview(filePreviewParameters: FilePreviewParameters): v
     filePreviewParameters.viewerAction,
     filePreviewParameters.fileOpenPreference,
     filePreviewParameters.conversationId,
+    filePreviewParameters.sizeInBytes,
```

Comment:
@maglims @AE-MS  need to add this is the array end in order prevent array indexing. Because Teams client is reading file title, url etc based on array index and if we add this in the middle it will change all indexes.